libs/components/indicators/src/lib/modules/icon/icon.component.ts

```diff
@@ -30,13 +30,13 @@ export class SkyIconComponent {
 
   /**
    * The size of the icon using
-   * [Font Awesome sizes](https://fontawesome.com/how-to-use/on-the-web/styling/sizing-icons). Do not prefix the size with `fa-`.
+   * [Font Awesome sizes](https://fontawesome.com/v4/examples/). Do not prefix the size with `fa-`.
    */
   @Input()
   public size: string | undefined;
 
   /**
-   * Whether the icon has a fixed width.
+   * Ensures all icons of the same size take up the same width. Useful for stacking icons vertically.
```

Comment:
our current doc is not descriptive enough as to what fixed width actually does, todo: get confirmation from John Lyons on wording

Reply:
Good idea. Thoughts @blackbaud-johnly?

Reply:
How about:

"Whether to enforce a fixed width based on icon size. By default, icons of a specified size share a consistent height, but their widths vary and can throw off vertical alignment. Use a fixed width when you stack icons vertically, such as in lists."